examples/default/Makefile

```diff
@@ -34,10 +34,10 @@ USEMODULE += ps
 # include and auto-initialize all available sensors
 USEMODULE += saul_default
 
-BOARD_PROVIDES_NETIF := airfy-beacon cc2538dk fox iotlab-m3 iotlab-a8-m3 mulle \
-        microbit native nrf51dongle nrf52dk nrf6310 openmote-cc2538 pba-d-01-kw2x \
-        remote-pa remote-reva samr21-xpro \
-        spark-core telosb yunjia-nrf51822 z1
+BOARD_PROVIDES_NETIF := acd52832 airfy-beacon cc2538dk fox iotlab-m3 iotlab-a8-m3 \
```

Comment:
Isn't possible for this board to also use the softdevice? I guess #8068 has good chances to be merged soon which brings back the softdevice compatibility.

Reply:
Generally it should be possible. But to be honest, currently the automatic inclusion of the softdevice is a pain in the neck. Let's include it once fixed.

Reply:
Ok, I might take care of that.